contrib_bots/lib/john.py

```diff
@@ -0,0 +1,132 @@
+import json
+import os
+import time
+import sys
+
+from random import choice
+
+try:
+    from chatterbot import ChatBot
+    from chatterbot.trainers import ChatterBotCorpusTrainer, ListTrainer
+except ImportError:
+    raise ImportError("""It looks like you are missing chatterbot.
+                      Please: pip install chatterbot""")
+
+CONTRIB_BOTS_DIR = os.path.dirname(os.path.abspath('zulip/contrib_bots'))
+os.chdir(os.path.dirname(CONTRIB_BOTS_DIR))
+
+JOKES_PATH = os.path.expanduser('lib/John/var/jokes.json')
```

Comment:
Hmm, I don't think you need to call `expanduser` since you are not including `~` in your path.